docs/authoring-json-rules.md

```diff
@@ -222,10 +222,10 @@ Example:
 }
 ```
 
-#### **In [[*]](#note)**
+#### **In**
 *Type: array of basic JSON values (integer, float, string, bool, null)*
 
-Evaluates the value of the `path` in the template using the `equals` operator for each value specified in the array.  If any results in `true`, `in` will evaluate to true; `false` otherwise.  All values in the array must be of the same type.
+Evaluates the value of the `path` in the template using the `equals` operator for each value specified in the array.  If any results in `true`, `in` will evaluate to true; `false` otherwise.
```

Comment:
It would be helpful to add a small statement that different types within the array *are* allowed.